dist/main.js

```diff
@@ -0,0 +1,79 @@
+let render = new Renderer()
+let board
+let rows
+let columns 
+
+$("button").on("click", ()=>{
+    rows = $("#rows").val()
+    columns= $("#columns").val()
+    board = new GoldRush(rows, columns)
+    board.loadBoard()
+    render.renderBoard(board.matrix)
+    $("#matrix").css("grid-template-rows", `repeat(${rows}, 1fr)`)
+    $(".matrix").css("grid-template-columns", `repeat(${columns}, 1fr`)
+    render.renderScore1(board.players[1].score)
+    render.renderScore2(board.players[0].score)   
+})
+
+$(document).keypress(function (e) {
+    switch (e.which) {
+        case 105:
+            board.movePlayer(1, "up")
+            render.renderBoard(board.matrix)
+            $("#matrix").css("grid-template-rows", `repeat(${rows}, 1fr)`)
```

Comment:
you are repeating this jQuery a lot. Try to think of a way to make a function that you can use instead of copying and pasting these two lines in each switch statment